dlna/dms/cms-desc.go

```diff
@@ -0,0 +1,185 @@
+package dms
```

Comment:
It might help to rename this file back to cm-service-desc.go, to reduce the diff.